src/_css/LoginComponent.css

```diff
@@ -88,6 +88,9 @@ input,
   border: 0;
   padding-left: 33%; }
 
+.ui.basic.table tbody tr {
```

Comment:
Please remove this file. We are supposed to be using `.scss` not `.css`. It had been deleted by Hawi but unfortunately, Faith added it again

Reply:
Well noted.